metatron-app/metatron-discovery
# fix admin users and workspace

### Description
fix admin users and workspace

**Related Issue** : 

METATRON-2881
METATRON-2885
METATRON-2888
METATRON-2920

### How Has This Been Tested?

METATRON-2881
1. Go to workspace management detail
2. Check "Set as the favorite"

METATRON-2885
1. Go to the permission schema list
2. Set the start time in the Duration Settings area
3. Select Apply button without setting end time
4. See the result without error

METATRON-2888
1. Go to permission schema detail
2. Select User Role Settings button
3. Check if Delete button does not exist in Default Role.

METATRON-2920
1. Add a member
2. Create a user whose username is less than 3 or more than 20 characters.
3. See the error message


#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
